Marlin/src/module/motion.cpp

```diff
@@ -1602,25 +1602,26 @@ void homeaxis(const AxisEnum axis) {
       do_homing_move(axis, -ABS(backoff[axis]) * axis_home_dir, homing_feedrate(axis));
   #endif
 
-  do_homing_move(axis, 1.5f * max_length(TERN(DELTA, Z_AXIS, axis)) * axis_home_dir);
-
-  #if BOTH(HOMING_Z_WITH_PROBE, BLTOUCH_SLOW_MODE)
-    if (axis == Z_AXIS) bltouch.stow(); // Intermediate STOW (in LOW SPEED MODE)
```

Comment:
Why is this moved?

Reply:
Need to know if a bump is requested BEFORE calling the first homing move, telling it NEED or NEED_NOT turn off the heaters. 

Actually this was not moved. What was moved is the block
```
  // Determine if a homing bump will be done and the bumps distance
  // When homing Z with probe respect probe clearance
  const bool use_probe_bump = TERN0(HOMING_Z_WITH_PROBE, axis == Z_AXIS && home_bump_mm(Z_AXIS));
  const float bump = axis_home_dir * (
    use_probe_bump ? _MAX(TERN0(HOMING_Z_WITH_PROBE, Z_CLEARANCE_BETWEEN_PROBES), home_bump_mm(Z_AXIS)) : home_bump_mm(axis)
  );
  ```
  But the `diff` shows it slightly otherwise.